students/fill0s0fear/hw-1/first.js

```diff
@@ -0,0 +1,25 @@
+let str = prompt('Enter a text');
+let maxRow = {
```

Comment:
Why do you use `let` here?

Reply:
I use let here because it's better then var. 
Currently, I understood that const will be better. Using const allow us to not change incoming text somehow

Reply:
so would you mind to change it to `const`?

Reply:
changed in new commit (3rd)